app/src/main/java/com/powerhungers/fomezero/feature/login/data/remote/data_source/LoginRemoteDataSource.kt

```diff
@@ -30,5 +30,7 @@ class LoginRemoteDataSource(private val service: LoginService) {
         cpf = cpf.orEmpty(),
         createdAt = createdAt.orEmpty(),
         updatedAt = updatedAt.orEmpty(),
+        token = token.orEmpty(),
+        refresh_token = refresh_token.orEmpty()
```

Comment:
```suggestion
        refreshToken = refreshToken.orEmpty()
```
o padrão de variaveis no android é camelCase e não snake_case como vem do backend